test/membrane_rtmp_plugin/rtmp_source_bin_test.exs

```diff
@@ -116,6 +116,35 @@ defmodule Membrane.RTMP.SourceBin.IntegrationTest do
     assert :ok = Task.await(ffmpeg_task)
   end
 
+  test "SourceBin gracefully handles start when starting after timestamp overflow" do
+    # offset half a second in the future
```

Comment:
nitpick: I think it would be better to start some more time in the future to avoid any potential accidental rounding errors done in the test when passing the offset around